ansible/ansible
# Initial ansible-test sanity docs.

##### SUMMARY

Initial ansible-test sanity docs:

- Add a dedicated documentation page for each `ansible-test sanity` command.
- Enforce the presence of documentation for each command in the sanity tests.

Once this documentation is deployed at a permanent location, we can do the following:

- Update ansible-test to include a docs URL for each bot test result file.
- Update the bot to include a link to error-specific docs in PR comments.

This should make the bot comments with error messages much more useful to contributors, as there will be documentation available for any given category of test. This PR provides only minimal documentation to get things started. Once deployed the documentation can be expanded to include the rationale for each test as well as detailed instructions on how to correct reported issues.

##### ISSUE TYPE

Feature Pull Request

##### COMPONENT NAME

ansible-test

##### ANSIBLE VERSION

```
ansible 2.4.0 (at-sanity-docs 5eba80e615) last updated 2017/07/13 18:37:49 (GMT -700)
  config file = None
  configured module search path = [u'/Users/mclay/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/mclay/code/mattclay/ansible/lib/ansible
  executable location = /Users/mclay/code/mattclay/ansible/bin/ansible
  python version = 2.7.11 (default, Jan 22 2016, 08:29:18) [GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)]
```
